NAMapKit/NATiledImageMapView.m

```diff
@@ -32,7 +32,7 @@ - (id)initWithFrame:(CGRect)frame tiledImageDataSource:(NSObject <ARTiledImageVi
 - (void)createImageView
 {
     if (self.dataSource) {
-        _imageView = [[ARTiledImageView alloc] initWithDataSource:self.dataSource];
+        self.imageView = [[ARTiledImageView alloc] initWithDataSource:self.dataSource];
```

Comment:
Is this right? I thought you're supposed to initialize this via `_` and then refer to it via `self`, but I am no expert.